lib/components/user/terms-of-use-pane.tsx

```diff
@@ -16,11 +16,15 @@ const TermsOfUsePane = ({
   disableCheckTerms,
   handleBlur,
   handleChange,
+  locale,
+  termsOfServiceLink,
```

Comment:
Please add `termsOfServiceLink` to the config types. I guess `termsOfStorageSet` needs to be added too.